kubernetes/check-pods.sh

```diff
@@ -0,0 +1,32 @@
+#! /bin/bash -e
+
+pod_label=$1
+statefulset=$2
+
+# Statefulset starts the pods individually, but with the convention `{statefulset-name}-{0-n}`
+# The second member {0-n} corresponds to the amount of pods
+# We have to manually defined name of the pods based of amount of replicas.
+if [[ "$statefulset" == "true" ]]; then
+    pods_name=()
+    for i in `seq 0 $(($(kubectl get statefulset $pod_label -o jsonpath='{.spec.replicas}')-1))`; do
```

Comment:
Please use a helper variable for readability:
``` bash
replica_count="$(kubectl get statefulset $pod_label --output jsonpath='{.spec.replicas}')"
```

Reply:
Done.